### ISSUE TITLE
corrupt mkl package?

### ISSUE THREAD

**Issue by janssen** [conda/conda#4410]
```
BUILD START: matplotlib-2.0.0rc1-py27_0
updating index in: C:\Miniconda2\conda-bld\win-64
updating index in: C:\Miniconda2\conda-bld\noarch

The following NEW packages will be INSTALLED:

    cycler:          0.10.0-py27_0 
    freetype:        2.5.5-vc9_2    [vc9]
    functools32:     3.2.3.2-py27_0
    libpng:          1.6.27-vc9_0   [vc9]
    mkl:             2017.0.1-0    
    msvc_runtime:    1.0.1-vc9_0    [vc9]
    numpy:           1.11.3-py27_0 
    pip:             9.0.1-py27_1  
    pyparsing:       2.1.4-py27_0  
    python:          2.7.13-0      
    python-dateutil: 2.6.0-py27_0  
    pytz:            2016.10-py27_0
    setuptools:      27.2.0-py27_1 
    six:             1.10.0-py27_0 
    vs2008_runtime:  9.00.30729.1-2
    wheel:           0.29.0-py27_0 
    zlib:            1.2.8-vc9_3    [vc9]

Traceback (most recent call last):
  File "C:\Miniconda2\Scripts\conda-build-script.py", line 5, in <module>
    sys.exit(conda_build.cli.main_build.main())
  File "C:\Miniconda2\lib\site-packages\conda_build\cli\main_build.py", line 321, in main
    execute(sys.argv[1:])
  File "C:\Miniconda2\lib\site-packages\conda_build\cli\main_build.py", line 312, in execute
    noverify=args.no_verify)
  File "C:\Miniconda2\lib\site-packages\conda_build\api.py", line 89, in build
    need_source_download=need_source_download, config=config)
  File "C:\Miniconda2\lib\site-packages\conda_build\build.py", line 1460, in build_tree
    config=config)
  File "C:\Miniconda2\lib\site-packages\conda_build\build.py", line 929, in build
    create_env(config.build_prefix, specs, config=config)
  File "C:\Miniconda2\lib\site-packages\conda_build\build.py", line 678, in create_env
    plan.execute_actions(actions, index, verbose=config.debug)
  File "C:\Miniconda2\lib\site-packages\conda\plan.py", line 837, in execute_actions
    execute_instructions(plan, index, verbose)
  File "C:\Miniconda2\lib\site-packages\conda\instructions.py", line 258, in execute_instructions
    cmd(state, arg)
  File "C:\Miniconda2\lib\site-packages\conda\instructions.py", line 119, in UNLINKLINKTRANSACTION_CMD
    txn.execute()
  File "C:\Miniconda2\lib\site-packages\conda\core\link.py", line 255, in execute
    self.verify()
  File "C:\Miniconda2\lib\site-packages\conda\core\link.py", line 247, in verify
    maybe_raise(CondaMultiError(exceptions), context)
  File "C:\Miniconda2\lib\site-packages\conda\exceptions.py", line 605, in maybe_raise
    raise CondaMultiError(non_clobber_errors)
conda.CondaMultiError: The package for mkl located at C:\Miniconda2\pkgs\mkl-2017.0.1-0
appears to be corrupted. The path 'Library/bin/mkl_vml_mc3.dll'
specified in the package manifest cannot be found.

The package for mkl located at C:\Miniconda2\pkgs\mkl-2017.0.1-0
appears to be corrupted. The path 'Library/bin/svml_dispmd.dll'
specified in the package manifest cannot be found.
```

**Comment by kalefranz** [comment]
I checked the package contents.
https://repo.continuum.io/pkgs/free/win-64/mkl-2017.0.1-0.tar.bz2

It appears to be in order.  It's possible something happened to the package in your package cache.

First, can you manually verify for me that `C:\Miniconda2\pkgs\mkl-2017.0.1-0\Library\bin
\mkl_vml_mc3.dll` and `C:\Miniconda2\pkgs\mkl-2017.0.1-0\Library\bin\svml_dispmd.dll` really don't exist?

Then, can you try

    conda clean --tarballs --packages

and make sure `C:\Miniconda2\pkgs\mkl-2017.0.1-0` is removed, then try again?

This bothers me a little bit.  I'd like to try to get to the bottom of it if we can.

**Comment by wjanssen** [comment]
Both of those DLLs do in fact exist in the location you indicate.  And the clean did fix the issue.  Could this be caused by multiple conda builds going on simultaneously on this machine?  Seems like they would both be side-effecting C:\Miniconda2\pkgs at the same time?  Is there a way to prevent that?

**Comment by kalefranz** [comment]
I think @msarahan is working with locking to prevent multiple conda builds interfering with each other, but I also think it's true that locking is a WIP and not perfect.  He might have further comment.

**Comment by msarahan** [comment]
This might also be a duplicate of #4374

Please ensure that only one build at a time is running, and report if this issue recurs.

**Comment by jhoblitt** [comment]
I have frequently seen the mkl package fail to install under osx in my CI env due to what I assume is a CDN problem.  Switching to a local conda channel mirror seems to have fixed the issue.

**Comment by kalefranz** [comment]
This has been fixed via #4789 and/or #5098.
